### ISSUE TITLE
Add aria-label to image links

### ISSUE THREAD

**Issue by m-vo** [contao/contao#2711]
Should we add `aria-label="<metadata alt>"` to the lightbox/image links we're rendering?

Chrome Lighthouse audit otherwise reports this as `Links do not have a discernible name`.

**Comment by ausi** [comment]
I’m not sure. Wouldn’t this read you the alt text twice with a screen reader then?

**Comment by aschempp** [comment]
I don't think so either. Also, why would the `aria-label` be the image description (which most likely is already present on the lower resolution image)? And what's the difference to a link `title` attribute?

**Comment by m-vo** [comment]
I don't know what's the right thing to do tbh. The lighthouse audit could very well be wrong here.

**Comment by leofeyer** [comment]
As discussed in Mumble on April 8th, adding the image description multiple times (in the `alt` attribute, the `title` attribute, the image caption and the `aria-label` attribute) is an anti-pattern, because screen readers will read it multiple times. Using the `alt` attribute should be compatible with all screen readers and therefore enough.

**Comment by m-vo** [comment]
Seems this was a lighthouse false positive. It's not reported anymore, now.
